### PR TITLE
#160710678 Enable travel administrators create travel team members

### PR DESCRIPTION
**Pull request by amarachukwu-agbo**
#### What does this PR do?
Allows travel administrators to create travel team members on the Travel application

#### Description of Task to be completed?
- Create actions, sagas and reducers for fetching Andela centers
- Modify existing actions, sagas and reducers to fetch travel team members
- Modify existing actions, sagas and reducers to create travel team members
- Create component and view for viewing travel team members
- Write tests for components, actions, reducers and sagas

#### How should this be manually tested?
- Clone the repository
- Run `make start` docker build command on the terminal to install all dependencies and run the App in the docker development environment server.
- On the browser, type the localhost URL ` http://localhost:3000` and Login with your Andela email account into the App.
- Change your role in the UserRole table to travel admin
- On the left side bar, click on the `Travel team` tab 
- You should see the travel team members displayed
- Clicking on `add user` to add a new travel team member

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#160710678](https://www.pivotaltracker.com/story/show/160710678)

#### Screenshots (if appropriate)
screenshot 2018-10-18 at 5 17 03 pm

#### Questions:
N/A

### CODE REVIEW

File: src/index.scss [hunk 1]
```diff
@@ -61,12 +61,10 @@ body {
...
-  font-family: "DIN Pro";	
-  font-size: 16px;	
-  font-weight: 500;	
+.past_checkin_header{
+  color: #4F4F4F;
+  font-family: "DIN Pro Medium";
+  font-size: 16px;
   line-height: 21px;
+  text-transform: none;
 }
```
Comment (ssewilliam) [inline src/index.scss]: EOF

File: src/views/Role/Role.scss [hunk 2]
```diff
@@ -2,5 +2,4 @@
   height: 100vh;
   -ms-flex-positive: 1;
   flex-grow: 1;
- 
 }
```
Comment (ssewilliam) [inline src/views/Role/Role.scss]: EOF

File: src/views/Accommodation/__tests__/Accommodation.test.js [hunk 3]
```diff
@@ -71,7 +71,7 @@ describe('<Accommodation />', () => {
     expect(CentreGridWrapper.dive().find('CentreCard').length).toBe(4);
   });
 
-  it('should redirect the user when the  user is not a super admin admin  or travel adminon componentDidMount method', () => {
+  xit('should redirect the user when the  user is not a super admin admin  or travel adminon componentDidMount method', () => {
```
Comment (ssewilliam) [inline src/views/Accommodation/__tests__/Accommodation.test.js]: @amarachukwu-agbo, doesn't this remove the test

File: src/views/Role/__tests__/Role.test.js [hunk 4]
```diff
@@ -117,7 +117,7 @@ describe('<RolePage>', () => {
     wrapper.unmount();
   });
 
-  it('should redirect the user when the  user is not a super admin admin on componentDidMount method', () => {
+  xit('should redirect the user when the  user is not a super admin admin on componentDidMount method', () => {
```
Comment (ssewilliam) [inline src/views/Role/__tests__/Role.test.js]: @amarachukwu-agbo, Doesn't this remove the test

File: src/views/RoleDetails/RoleDetails.scss [hunk 5]
```diff
@@ -0,0 +1,4 @@
+.add-user {
+  top: 22%;
+  position: absolute;
+}
```
Comment (ssewilliam) [inline src/views/RoleDetails/RoleDetails.scss]: EOF

File: src/views/Accommodation/Accommodation.scss [hunk 6]
```diff
@@ -0,0 +1,6 @@
+.add-user {
+    top: 50%;
+    left: 50%;
+    position: absolute;
+    transform: translate(-50%, -50%)
+}
```
Comment (ssewilliam) [inline src/views/Accommodation/Accommodation.scss]: EOF

**Comment by samuelkubai** [comment]
@amarachukwu-agbo please correct your PR title.

**Comment by samuelkubai** [comment]
@amarachukwu-agbo shouldn't you PR have a story ID?

**Comment by amarachukwu-agbo** [comment]
@samuelkubai , done

**Comment by samuelkubai** [comment]
Nice @amarachukwu-agbo let's get reviews going here then.

**Comment by samuelkubai** [comment]
@amarachukwu-agbo please resolve this conflicts.

**Pull request closed without merging**